iree/compiler/Dialect/Flow/Transforms/OutlineDispatchRegions.cpp

```diff
@@ -214,7 +214,8 @@ static LogicalResult outlineDispatchWorkgroupsOp(
   // Add executable entry point pointing at the function.
   OpBuilder builder(executableOp.body());
   auto entryPointOp = builder.create<DispatchEntryOp>(
-      regionOp.getLoc(), builder.getStringAttr(workgroupFuncOp.getName()),
+      regionOp.getLoc(), /*sym_visibility=*/nullptr,
+      builder.getStringAttr(workgroupFuncOp.getName()),
```

Comment:
@benvanik any preference for how to avoid this extra arg for the optional visibility attribute? Add/edit a custom builder? I'm hitting the same thing with the HAL ops.

Reply:
I think custom builders are the way - which may be worth it for these ops - this is fine for now (unless you want to add some sensible ones too so that for example we could just pass workgroupFunOp.getName() instead of having to wrap it in an attr)

Reply:
Added a builder for this op.